src/parser/classes/livechat/items/LiveChatSponsorshipsHeader.ts

```diff
@@ -0,0 +1,31 @@
+import { Parser } from '../../../index.js';
+import { YTNode } from '../../../helpers.js';
+import type { ObservedArray } from '../../../helpers.js';
+import type { RawNode } from '../../../index.js';
+import NavigationEndpoint from '../../NavigationEndpoint.js';
+import Text from '../../misc/Text.js';
+import Thumbnail from '../../misc/Thumbnail.js';
+import LiveChatAuthorBadge from '../../LiveChatAuthorBadge.js';
+
+export default class LiveChatSponsorshipsHeader extends YTNode {
+  static type = 'LiveChatSponsorshipsHeader';
+
+  author_name: Text;
+  author_photo: Thumbnail[];
+  author_badges: ObservedArray<LiveChatAuthorBadge> | null;
+  primary_text: Text;
+  menu_endpoint: NavigationEndpoint;
+  context_menu_accessibility_label: string;
+  image: Thumbnail[];
+
+  constructor(data: RawNode) {
+    super();
+    this.author_name = new Text(data.authorName);
+    this.author_photo = Thumbnail.fromResponse(data.authorPhoto);
+    this.author_badges = Parser.parse(data.authorBadges, true, LiveChatAuthorBadge);
```

Comment:
Could you please use parseArray here. I'm sorry for all the changes and code review comments.
```suggestion
    this.author_badges = Parser.parseArray(data.authorBadges, LiveChatAuthorBadge);
```